### PR TITLE
Introducing support for extra cases of llvm5 binaries for builtin_llvm=OFF

### PR DESCRIPTION
**Pull request by oshadura**

### CODE REVIEW

File: interpreter/CMakeLists.txt [hunk 1]
```diff
@@ -286,8 +286,10 @@ else()
   include(VersionFromVCS)
 
   set(PACKAGE_VERSION "${LLVM_PACKAGE_VERSION}")
-  if (NOT (PACKAGE_VERSION VERSION_EQUAL 5.0 OR PACKAGE_VERSION VERSION_EQUAL 5.0.1))
-    message(FATAL_ERROR "LLVM version different from 5.0 or 5.0.1!")
+  if (${PACKAGE_VERSION} MATCHES "5\\.0(|\\.[1-2])")
```
Comment (vgvassilev) [inline interpreter/CMakeLists.txt]: 5.0.x should be API compatible. Let's do .*

Comment (oshadura) [reply interpreter/CMakeLists.txt]: thanks, it is good remark! I forgot about ROOT patched version which is 5.0.0! Fixed!

**Comment by oshadura** [comment]
@phsft-bot build!

**Comment by oshadura** [comment]
@phsft-bot build!

**Comment by oshadura** [comment]
Build is actually with status OK
```
22:23:17 Build failed because: org.jenkinsci.plugins.workflow.job.WorkflowRun
```

**Pull request merged**